recipe/meta.yaml

```diff
@@ -34,35 +31,42 @@ requirements:
     - {{ compiler('cxx') }}
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
-    - patch     # [not win]
-    - m2-patch  # [win]
+
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.38.*
+    - llvmlite 0.39.*
     - numpy
-    - tbb-devel 2021.*
-    - _openmp_mutex            # [linux and not ppc64le]
+    - tbb-devel 2021.*,<2021.6
+    - _openmp_mutex            # [linux]
+    - importlib_metadata       # [py<39]
+
   run:
     - python
-    - {{ pin_compatible('llvmlite', max_pin='x.x') }}
-    - {{ pin_compatible('numpy') }}
+    - importlib_metadata       # [py<39]
+    - llvmlite >=0.39.*,<1
```

Comment:
should be llvmlite >=0.39.*,<0.40
https://github.com/numba/numba/blob/0.56.3/setup.py#L28

Reply:
Thats right, thanks!